### ISSUE TITLE
[Bug] Persistent error calling API endpoint GET /api/auth/user

### ISSUE THREAD

**Issue by jskillen** [thingsboard/thingsboard#3445]
**Describe the bug**
When calling the API endpoint GET /api/auth/user one would expect to have a payload returned with the user information. Instead I receive a 401 response with this payload:
```
{
  "status": 401,
  "message": "Invalid username or password",
  "errorCode": 10,
  "timestamp": "2020-09-10T23:55:35.448+0000",
  "subscriptionErrorCode": null,
  "subscriptionEntry": null,
  "subscriptionValue": null
}
```

**Your Server Environment**

* own setup
  * Azure cloud setup
  * ThingsBoard Version 3.1.1
  * Ubuntu 20

**Your Client Environment**

**Desktop (please complete the following information):**

* OS: Mac Catalina v 10.15.6
* Browser: Chrome version 85.0.4183.83

**To Reproduce**
1. Use the API to obtain a JWT token
2. With that token, use curl to call the /api/auth/user endpoint
3. Receive 401 error response with "Invalid username or password" message
4. For fun, try other API functions. Observe that they perform as expected.

**Expected behavior**
I expect the API to return a JSON payload of the current user.

**Additional context**
As mentioned, other API calls work as expected. Furthermore, I have found that the userService service does not contain the getCurrentUser() function.

**Comment by yefimov-andrey** [comment]
How did you use the token? Can you reproduce the error on cloud.thingsboard.io ?

**Comment by jskillen** [comment]
The token is used in the headers - I cribbed it from the swagger (which also doesn't work).
curl -X GET --header 'Accept: application/json' --header 'X-Authorization:...' 'https://myurl/api/auth/user'

I have my own PE installation so I did not try this on cloud.thingsboard.io. 

I have created an account there on cloud.thingsboard.io and I get the same results.

**Comment by yefimov-andrey** [comment]
Do you put "Bearer" before the token?

**Comment by jskillen** [comment]
Ah, @yefimov-andrey adding Bearer works. Thank you.

Strangely, not all endpoints need that text; furthermore, the swagger does not add it into the curl request.

**Comment by yefimov-andrey** [comment]
Some endpoints don't need a token, but ones that do, require "Bearer" with the token.

**Comment by jskillen** [comment]
@yefimov-andrey would it be prudent to update the swagger to reflect this? I can open another ticket for that.

**Comment by yefimov-andrey** [comment]
To reflect what? Which API calls don't require a token and which ones do?

**Comment by jskillen** [comment]
Yes, which do and which don't. And/or update the swagger to inject "Bearer " into the token value.

**Comment by yefimov-andrey** [comment]
This may be done in future releases.

**Comment by jskillen** [comment]
That would be appreciated. The swagger UI often does not work - it doesn't add "bearer" when it is required and includes the refreshtoken in the curl commands, which causes errors.
